src/Storefront/Resources/views/storefront/page/account/order-history/order-item.html.twig

```diff
@@ -65,7 +65,7 @@
                                 {# @deprecated tag:v6.7.0 - Will use Bootstrap `btn-light` class instead of custom styling #}
                                 <button class="btn order-table-header-context-menu{% if feature('ACCESSIBILITY_TWEAKS') %} btn-light float-end{% else %} dropdown-toggle{% endif %}"
                                         type="button"
-                                        id="accountOrderDropdown"
+                                        id="accountOrderDropdown-{{ order.id }}"
```

Comment:
This makes using an id unnecessary. Is there any reason why we shouldn't replace this by a simple class?

Reply:
In this case the `id` is not used for styling or so, but only for the `aria-labelledby`. If that can be removed (from the `.order-table-header-context-menu-content` element), then it could be removed completely.

Reply:
It is not 100% needed, but imo it doesn't hurt to make the dropdown be labelled by the button. Then the screen reader announces that the list of items (followed by the button after expanding it) belongs to the toggle button when selecting it. `"Actions, group"` instead of just `"group"` or `"list"`